test/expected/plan_ordered_append.out

```diff
@@ -0,0 +1,237 @@
+         ->  Index Scan using _hyper_1_1_chunk_ordered_append_time_idx on _hyper_1_1_chunk
+(5 rows)
+
+-- test ASC for reverse ordered chunks
+:PREFIX SELECT
+  time, device_id, value
+FROM ordered_append_reverse
+ORDER BY time ASC LIMIT 1;
+                                                 QUERY PLAN                                                 
+------------------------------------------------------------------------------------------------------------
+ Limit
+   ->  Append
+         ->  Index Scan Backward using _hyper_2_6_chunk_ordered_append_reverse_time_idx on _hyper_2_6_chunk
+         ->  Index Scan Backward using _hyper_2_5_chunk_ordered_append_reverse_time_idx on _hyper_2_5_chunk
+         ->  Index Scan Backward using _hyper_2_4_chunk_ordered_append_reverse_time_idx on _hyper_2_4_chunk
+(5 rows)
+
+-- test DESC for reverse ordered chunks
+:PREFIX SELECT
+  time, device_id, value
+FROM ordered_append_reverse
+ORDER BY time DESC LIMIT 1;
+                                            QUERY PLAN                                             
+---------------------------------------------------------------------------------------------------
+ Limit
+   ->  Append
+         ->  Index Scan using _hyper_2_4_chunk_ordered_append_reverse_time_idx on _hyper_2_4_chunk
+         ->  Index Scan using _hyper_2_5_chunk_ordered_append_reverse_time_idx on _hyper_2_5_chunk
+         ->  Index Scan using _hyper_2_6_chunk_ordered_append_reverse_time_idx on _hyper_2_6_chunk
+(5 rows)
+
+-- test query with ORDER BY column not in targetlist
+:PREFIX SELECT
+  device_id, value
+FROM ordered_append
+ORDER BY time ASC LIMIT 1;
+                                             QUERY PLAN                                             
+----------------------------------------------------------------------------------------------------
+ Limit
+   ->  Append
+         ->  Index Scan Backward using _hyper_1_1_chunk_ordered_append_time_idx on _hyper_1_1_chunk
+         ->  Index Scan Backward using _hyper_1_2_chunk_ordered_append_time_idx on _hyper_1_2_chunk
+         ->  Index Scan Backward using _hyper_1_3_chunk_ordered_append_time_idx on _hyper_1_3_chunk
+(5 rows)
+
+-- queries without LIMIT shouldnt use ordered append
+:PREFIX SELECT
+  time, device_id, value
+FROM ordered_append
+ORDER BY time ASC;
+                                          QUERY PLAN                                          
+----------------------------------------------------------------------------------------------
+ Merge Append
+   Sort Key: ordered_append."time"
+   ->  Index Scan Backward using ordered_append_time_idx on ordered_append
+   ->  Index Scan Backward using _hyper_1_1_chunk_ordered_append_time_idx on _hyper_1_1_chunk
+   ->  Index Scan Backward using _hyper_1_2_chunk_ordered_append_time_idx on _hyper_1_2_chunk
+   ->  Index Scan Backward using _hyper_1_3_chunk_ordered_append_time_idx on _hyper_1_3_chunk
+(6 rows)
+
+-- queries without ORDER BY shouldnt use ordered append
```

Comment:
this does use an append node so this comment is a bit weird

Reply:
its not an ordered append though but without an ORDER BY the planner will choose an append, the ordered append code won't put the parent into the list of append nodes thats how I can tell

Reply:
i added an comment there, or do you think i should remove this query?